### ISSUE TITLE
Create project card for BallotNav

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/website#703]
### Overview
Provide collateral for the HackforLA website

### Action Items
- [x]   Gather items
  - [x]  600 x 400 image
     - [x]  Alt image text
  - [x] 1500 x 700 hero image (please do not put project title on hero image)
     - [x]  Alt image text
  - [x]  Name of project
  - [x]  A blurb about your project
  - [x]  Links (github, slack channel url, Testing Site, Live Site, etc.)
  - [x]  Any resources for a Getting Started link (a link to a wiki, readme(s) or both)
  - [x]  What you are looking for skills wise
  - [x]  Partner(s)
  - [x]  Location
  - [x]  Status
- [ ]   Add project Card to website

### Resources/Instructions
See project cards on the hackforla.org website for examples

**Comment by Arjayellis** [comment]
600 x 400 image
 Alt image text
 1500 x 700 hero image (please do not put project title on hero image)
 Alt image text
 Name of project: BallotNav
 A blurb about your project: 
 Links (github, slack channel url, Testing Site, Live Site, etc.) 
- GitHub: https://github.com/hackforla/ballotnav
- Slack: https://hackforla.slack.com/archives/C0191KE1XB7 
 Any resources for a Getting Started link (a link to a wiki, readme(s) or both)
- Readme: https://github.com/hackforla/ballotnav/blob/master/README.md 
 What you are looking for skills wise
- Business Analyst (Category: BA)
- UI Designer (Category: UI/UX) 
- UX Researcher (Category: UI/UX) 
 Partner(s): Code for America, Other Brigades 
 Location: Remote
 Status: Active

**Comment by Arjayellis** [comment]
@myastark I wrote a draft for the blurb on the project card. Would love your expertise before submitting. 

BallotNav aims to support absentee voters by addressing limitations caused by conflicting information from the media, hard to navigate government websites, and statewide regulations, with an efficient way to get accurate information about where, when and how to deliver your ballot in person.

This will take dedicated partners across the country to keep this resource up to date by daily monitoring of Secretary of State and county registrar websites across all 50 states and 3,007 counties through November 3rd (X number of days left). The end result will be a clear path to action for those who need it most.

This task is nearly impossible, but no one is coming. It’s up to us to save our democracy, and we can do it.

**Comment by Arjayellis** [comment]
New draft based on @myastark suggestion. 

BallotNav aims to support absentee voters who are concerned about the present reliability of the USPS by addressing limitations caused by conflicting information from the media, hard to navigate government websites, and statewide regulations, with an efficient way to get accurate information about where, when and how to deliver your ballot in person.

This will take dedicated partners across the country to keep this resource up to date by daily monitoring of Secretary of State and county registrar websites across all 50 states and 3,007 counties through November 3rd (X number of days left). The end result will be a clear path to action for those who need it most.

This task is nearly impossible, but no one is coming. It’s up to us to save our democracy, and we can do it.

**Comment by ExperimentsInHonesty** [comment]
![boy at ballot box-600x400](https://user-images.githubusercontent.com/37763229/90967783-292a7900-e499-11ea-8662-889a803c7e5d.jpg)
Alt Image Text: Young boy at ballot drop off box title: Giving democracy a helping hand (CC BY-NC-SA 2.0) by eagle.dawg"

**Comment by ExperimentsInHonesty** [comment]
@Excursionpreneur - This project is fairly urgent.  Can you provide a progress update?

1. Progress
2. Blocks
3. Availability
4. ETA

**Comment by Arjayellis** [comment]
Removed Technical Architect from roles needed

**Comment by Excursionpreneur** [comment]
I made the card on my local server, and will need to verify the text that goes into the card.  For some reason, my code kept being deleted on my local server when I tried to update index.html with the new card, so testing it was difficult.

**Comment by jbubar** [comment]
@Excursionpreneur Instructions for finding the project id and a template for building the project card https://github.com/hackforla/website/wiki/Template-of-a-project.md-file

Handy work of Cynthia. Im just sharing the link

**Comment by kcoronel** [comment]
Hello @Excursionpreneur any updates on this? Thank you!

**Comment by alexandrastubbs** [comment]
PR: https://github.com/hackforla/website/pull/724

**Comment by alexandrastubbs** [comment]
@Excursionpreneur please put a screenshot of the project card from your computer into this issue, so that the stakeholder can perform their user acceptance testing.

**Comment by KianBadie** [comment]
I noticed that the repository name or id number was not included in out [data](https://github.com/hackforla/website/blob/gh-pages/_data/github-data.json) set. I'm guessing it is because it is not tagged hack for la. We fetch a repository that is untagged (work for la), so if ballot nav did not want to tag their repo with the needed tags, we would need to add that repo id to the [data fetching script](https://github.com/hackforla/website/blob/925d17c1a1286261a56fed5de42ce3a842b1d527/github-actions/get-project-data.js#L220)(I linked the line where we declare which untagged repos manually fetch). I think that is why the contributors section is blank. Because there is no data in our data set matching the repo id in the project card. 

If we did update our script, we would have to wait until tomorrow for data to show up (since the script runs nightly). Either that or manually insert the data today a long with the script changes, which is doable I think but has a higher chance for error/mistakes

![Screenshot from 2020-09-12 10-23-25](https://user-images.githubusercontent.com/18221058/93001400-969f5780-f4e3-11ea-9791-95f36a0405f7.png)
![Screenshot from 2020-09-12 10-27-45](https://user-images.githubusercontent.com/18221058/93001401-9737ee00-f4e3-11ea-9905-9999705c91b6.png)
![Screenshot from 2020-09-12 10-27-56](https://user-images.githubusercontent.com/18221058/93001402-97d08480-f4e3-11ea-91d0-7fbf3f4ca788.png)
![Screenshot from 2020-09-12 10-28-06](https://user-images.githubusercontent.com/18221058/93001403-98691b00-f4e3-11ea-8869-c3f09f751e60.png)

**Comment by ExperimentsInHonesty** [comment]
We need the leadership info for this project
Name
Role
Github handle
slack member id

**Comment by ExperimentsInHonesty** [comment]
**This is the format**
leadership:
  - name: Mya Stark
    role: 
    links: 
      slack: 'https://app.slack.com/team/UMC7AJLDV'
      github: 'https://github.com/myastark'
    picture: 'https://avatars.githubusercontent.com/myastark'

**Comment by Excursionpreneur** [comment]
The card was updated with "project leadership". The role and links were blank, so this commit is blank for now. 

![screencapture-0-0-0-0-4000-2020-09-13-18_02_03](https://user-images.githubusercontent.com/24662537/93033541-85da0900-f5eb-11ea-858c-029548094604.png)

Link to pull request - https://github.com/hackforla/website/pull/724

**Comment by Excursionpreneur** [comment]
Also, here is a screenshot of the page for BallotNav

![screencapture-0-0-0-0-4000-projects-ballot-nav-2020-09-13-18_06_15](https://user-images.githubusercontent.com/24662537/93033628-e79a7300-f5eb-11ea-9255-99d269e925fd.png)

**Comment by Excursionpreneur** [comment]
The code was pushed to the live site by @KianBadie. After discussing the text that should go into the card, it was decided that the text provided by Mya in the comments is what is expected (not what's in the README, as I presumed and submitted earlier). 

![screencapture-hackforla-org-2020-09-15-21_03_23](https://user-images.githubusercontent.com/24662537/93291061-fffac100-f796-11ea-9e0b-00900f7348c3.png)

![screencapture-hackforla-org-2020-09-15-21_03_23](https://user-images.githubusercontent.com/24662537/93291094-0f7a0a00-f797-11ea-9220-5dcf25d5c190.png)

**Comment by KianBadie** [comment]
The BallotNav card has been merged and is viewable on the site. Even though the @Excursionpreneur received a visual confirmation from @myastark, we updated the description to match the one provided [here](https://github.com/hackforla/website/issues/703#issuecomment-678710811) in this issue. Because there was already a visual confirmation before taking into account the information provided here, it was not completely clear which information was preferred to have on the card. @Excursionpreneur will reach out to BallotNav team to see if any changes will be requested now that it is merged.

A project card image was also provided [here](https://github.com/hackforla/website/issues/703#issuecomment-678711114) in this issue, but because of the visual confirmation with the map image being used for both the hero and project card image, we decided to merge it how it was and see if any changes would like to be requested.  

Lastly, we believe this might have to do more with the VRMS api, but we noticed the word undefined showing up both on the project meetings page and this page. Will open a new issue, troubleshoot, and contact VRMS team to investigate. Link to issue [here](https://github.com/hackforla/website/issues/735)
![Screenshot from 2020-09-16 12-09-58](https://user-images.githubusercontent.com/18221058/93381498-9bcb1200-f815-11ea-99a6-7824abe75cd2.png)

**Comment by myastark** [comment]
![ballotnav-pr-image-web](https://user-images.githubusercontent.com/54009156/93534111-4a944e80-f8f9-11ea-92a9-9b899fe6caae.jpg)

**Comment by KianBadie** [comment]
@myastark Thank you for the image. I'm assuming this is the image you would like to use for the project card, while keeping the u.s map as the hero image? If so, did everything else on the page look acceptable? We got rid of the `undefined` behavior I mentioned in my [last](https://github.com/hackforla/website/issues/703#issuecomment-693609080) comment.

**Comment by myastark** [comment]
@KianBadie Yes, for the project card. Keeping the map for the hero is fine, but is it the right size? or was that just a problem with the card

**Comment by KianBadie** [comment]
@myastark sounds good. I can add those changes to the site within the hour. I just need an alt-text for the image first. 

The map image is the correct size 1500x700 pixels. When you say "or was that just a problem with the card", did you see something in the images of the page above that looked incorrect with the image or did I interpret that incorrectly?

**Comment by Arjayellis** [comment]
@KianBadie Are we able to change the "Looking for" section to "Data Scientists, UX Researchers, UI Designers, and anyone who wants to help us validate website data!"?

**Comment by KianBadie** [comment]
@Arjayellis Yeah I can change it to that :+1: Would I be able to wait until tomorrow to change that (unless it is urgent) so that I can add both your change and @myastark's change in the same commit. I am waiting on @myastark's response for the alt-text for the image she provided and to see if she had any other questions about the images.

Did you see anything else you would like to change @Arjayellis? Or is it just the "Looking For"

**Comment by ExperimentsInHonesty** [comment]
@KianBadie Alt text is "Person holding phone with ballot nav website on it".  Randall AKA Arjayellis said everything is good.

**Comment by KianBadie** [comment]
@myastark @Arjayellis @ExperimentsInHonesty Updated project image/alt-text, and updated the looking for. I changed the wording of the looking for to be singular to stay consistent with the site (eg: Researchers -> Researcher) is that ok?

Also, I made it so that once the updates are merged, it will close this issue as it seems like there are no other requested changes.

Lastly, we noticed in the description the segment "(X number of days left)" in the second paragraph. We are assuming this is a counter that will change each day. Unfortunately, with the current framework we are using (Jekyll) there is no straightforward way to display content like this dynamically. I wish it was as simple as adding a little JavaScript, but the Jekyll framework adds a bit more hoops that would take some further digging to figure out how to get around. Would you like us to remove that phrase? Or would you like to keep it in the description?

![Screenshot from 2020-09-18 10-15-58](https://user-images.githubusercontent.com/18221058/93626467-5125d300-f998-11ea-8872-3a82a36699da.png)
![Screenshot from 2020-09-18 10-16-25](https://user-images.githubusercontent.com/18221058/93626469-51be6980-f998-11ea-8d20-680c80accd20.png)
![Screenshot from 2020-09-18 10-16-37](https://user-images.githubusercontent.com/18221058/93626472-52570000-f998-11ea-878c-67c0b379f776.png)
![Screenshot from 2020-09-18 10-16-53](https://user-images.githubusercontent.com/18221058/93626474-52570000-f998-11ea-866f-54ced58a732c.png)

**Comment by myastark** [comment]
we ended up killing the counter feature anyhow -i missed this update. @KianBadie please change text to:

BallotNav will help voters quickly find the location, times of operation, and due date for dropping off mail-in ballots in person. Due to COVID-19 and cuts to the United States Postal Service, some voters may feel insecure not only about going out to the polls, but also about utilizing the mail to cast their vote. While it is possible to drop off your mail-in ballot in person, the rules for doing so are not clearly outlined in one single location. Information is scattered across state and county websites, each of which may individually fail to provide one or more elements of crucial information, such as available ballot drop-off times. The BallotNav project will collect this data through a network of brigade partnerships and update it accordingly leading up to November’s election. 

BallotNav’s overall goal is to provide information that helps voters cast their ballots safely and correctly, and avoid accidental disenfranchisement.

Open Roles:
Community Data Monitors: We need an army of people to collect information about drop-off sites for 3,000 counties, and remain on-call to update it when changes occur.
Marketing, PR & Partnerships
Fundraising
Software Developers & Data Scientists
Translators (Spanish especially)

**Comment by KianBadie** [comment]
@myastark sounds good. Here is what the card now looks like (I just uploaded the parts that changed).
![Screenshot from 2020-09-19 13-19-47](https://user-images.githubusercontent.com/18221058/93688529-4b54ee00-fa7b-11ea-847b-c7b1134728fc.png)
![Screenshot from 2020-09-19 13-19-33](https://user-images.githubusercontent.com/18221058/93688527-4abc5780-fa7b-11ea-9441-f52e055ac398.png)
